msi/gs60/default.nix

```diff
@@ -0,0 +1,39 @@
+{ config, pkgs, lib, ... }:
+{
+  imports = [
+    ../../common/cpu/intel
+    ../../common/pc/laptop
+  ];
+
+  boot = {
+
+    # Interferes with Fn- action keys
+    kernelParams = [ "video.report_key_events=0" ];
+
+    # Workaround for problematic firmware loading
+    extraModprobeConfig = ''
+      options ath10k_core skip_otp=y
+    '';
+
+    initrd = {
+      availableKernelModules = [
+        "ahci"
+        "ehci_pci"
+        "rtsx_pci_sdmmc"
+        "sd_mod"
+        "usb_storage"
+        "xhci_pci"
+      ];
+    };
```

Comment:
Also this is correct, I think it is better to rely on information on nixos-generate hardware because it will be regenerated for new kernel versions. Otherwise this looks good to me:

```suggestion
```